docs/content/using/using_hostdefinition.md

```diff
@@ -0,0 +1,72 @@
+# Using dynamic host definition
+
+Dynamic host connectivity removes the necessity for manual host definitions. This is also facilitates the scaling out process of a node. 
+
+The `HostDefiner` identifies the hosts available for host definition on each storage system and controls each of the host definitions. To see the statuses of the managed host definitions by the host definer, use the `hostdefinition` command.
```

Comment:
is `kubectl` omitted from:
>the `get hostdefinitions` command

on purpose? because there isn't actually a command that would work by starting with `get`